elements/pfe-sass/mixins/_mixins.scss

```diff
@@ -55,49 +55,12 @@
-    --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--darkest--link)};
-    --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--darkest--link--visited)};
-    --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--darkest--link--hover)};
-    --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--darkest--link--focus)};
-  }
-  @else if $theme == accent {
-    --pfe-broadcasted--color--text:               #{pfe-color(surface--accent--text)};
-    --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--accent--link)};
-    --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--accent--link--visited)};
-    --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--accent--link--hover)};
-    --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--accent--link--focus)};
-  }
-  @else if $theme == complement {
-    --pfe-broadcasted--color--text:               #{pfe-color(surface--complement--text)};
-    --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--complement--link)};
-    --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--complement--link--visited)};
-    --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--complement--link--hover)};
-    --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--complement--link--focus)};
-  }
-
-  @else if $theme == lighter {
-    --pfe-broadcasted--color--text:               #{pfe-color(surface--lighter--text)};
-    --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--lighter--link)};
-    --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--lighter--link--visited)};
-    --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--lighter--link--hover)};
-    --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--lighter--link--focus)};
-  }
-  @else if $theme == lightest {
-    --pfe-broadcasted--color--text:               #{pfe-color(surface--lightest--text)};
-    --pfe-broadcasted--color--ui-link:            #{pfe-color(surface--lightest--link)};
-    --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(surface--lightest--link--visited)};
-    --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(surface--lightest--link--hover)};
-    --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(surface--lightest--link--focus)};
+  @else if $theme == saturated {
+    --pfe-broadcasted--color--text:               #{pfe-color(text--on-saturated)};
+    --pfe-broadcasted--color--ui-link:            #{pfe-color(ui-link--on-saturated)};
+    --pfe-broadcasted--color--ui-link--visited:   #{pfe-color(ui-link--on-saturated--visited)};
+    --pfe-broadcasted--color--ui-link--hover:     #{pfe-color(ui-link--on-saturated--hover)};
+    --pfe-broadcasted--color--ui-link--focus:     #{pfe-color(ui-link--on-saturated--focus)};
```

Comment:
@castastrophe this is what I was thinking about re: my question of this MR & the broadcast colors MR being somewhat intertwingled.  Just verifying: you still prefer to merge the broadcast-colors MR first?

Reply:
I still want to merge the broadcasted story first.